ravenframework/JobHandler.py

```diff
@@ -214,7 +215,7 @@ def __initializeRay(self):
           # add names in runInfo
           self.runInfoDict['remoteNodes'] = servers
           ## initialize ray server with nProcs
-          self.rayServer = ray.init(address=address, _redis_password=redisPassword,log_to_driver=False,include_dashboard=db) if _rayAvail else pp.Server(ncpus=int(nProcsHead))
```

Comment:
could you add an explanation about the remove of redis? it seems ray will no longer launch redis by default.

Reply:
ray start no longer uses redispassword so I removed it.

Reply:
In https://docs.ray.io/en/latest/ray-core/package-ref.html, the _redis_password is still present in the method, just by default, ray will not launch redis, am I right here? @joshua-cogliati-inl

Reply:
Except it is not present in https://docs.ray.io/en/latest/ray-core/package-ref.html#ray-start and ray start --head no longer outputs it.  This change also does not seem to be mentioned in the release notes, so I am at least somewhat mystified, but it does not seem to be used as much any more.

Reply:
The current usage of redis_password is in: https://github.com/ray-project/ray/blob/master/python/ray/util/collective/collective_group/gloo_collective_group.py#L71

For what it is worth, in ray 1.12, the usage of redis_password is:
```python
            redis_password = ray_constants.REDIS_DEFAULT_PASSWORD
            redisStore.authorize(redis_password)
```
that is, it is completely ignored.